### ISSUE TITLE
Angular Knobs changing property name to lower case

### ISSUE THREAD

**Issue by paulparton** [storybooks/storybook#2641]
### Issue details
When using knobs in Angular the property names are converted to lower case inside KnobWrapperComponent.knobChanged. This is making it impossible to use a knob on a property with a camel cased name

https://github.com/storybooks/storybook/blob/master/addons/knobs/src/angular/helpers.js#L76

### Steps to reproduce

When I console log the the ngOnChanges object for the following, the property name changed is "buttontype" instead of "buttonType"

```
storiesOf('My Story', module)
  .addDecorator(withKnobs)
  .add('Button types', () => {
    const types = [
      'default',
      'primary',
      'secondary'
    ];
    const label = select('buttonType', types, types[0]);
    return {
      component: ButtonComponent,
      props: {
      }
    };
  })

```
### Please specify which version of Storybook and optionally any affected addons that you're running

    "@storybook/addon-knobs": "^3.3.3",
    "@storybook/angular": "^3.3.3",

**Comment by ndelangen** [comment]
Is this behavior seems to not exist in React, I wonder why this was added..

Can you remember why this was added @alterx ?

**Comment by alterx** [comment]
It's probably a leftover from the first implementation (problems of having to duplicate code 😢 ), right now I can't think of a justification for this. Will look into it.

**Comment by paulparton** [comment]
Awesome! thanks for the fast response guys :)

**Comment by Hypnosphi** [comment]
Fix released as [`3.3.4`](https://github.com/storybooks/storybook/releases/tag/v3.3.4)
